cpp/src/gandiva/function_registry_test.cc

```diff
@@ -31,7 +31,8 @@ TEST_F(TestFunctionRegistry, TestFound) {
 
   const NativeFunction* function = registry_.LookupSignature(add_i32_i32);
   EXPECT_NE(function, nullptr);
-  EXPECT_EQ(function->signature(), add_i32_i32);
+  EXPECT_TRUE(std::find(function->signatures().begin(), function->signatures().end(),
```

Comment:
This should do the trick `EXPECT_THAT(add_i32_i32, In(function->signatures())`.

Reply:
using `Contains` matcher from gmock. couldn't find `In`.